modules/frontend/frontend-js-web/src/META-INF/resources/liferay/modules.js

```diff
@@ -946,5 +946,14 @@
 		useBrowserConsole: false
 	};
 
+	Loader.addModule(
```

Comment:
Hey guys, I saw we have a soyutils copy inside the `frontend-js-web` module. This is currently being used mainly by `ddm`.

I've reused that one for now, but I think we could create a different bundle and grab the file from `bower` as we do here and in the `babel-polyfill` module. What do you think?